src/include/cell.i

```diff
@@ -787,10 +864,14 @@ restart:
             break;
         flags = *p++; /* skip second descriptor byte */
 
+        if (LF_ISSET(WT_CELL_PREPARE))
+            F_SET(unpack, WT_CELL_UNPACK_PREPARE);
         if (LF_ISSET(WT_CELL_TS_START))
             WT_RET(__wt_vunpack_uint(&p, end == NULL ? 0 : WT_PTRDIFF(end, p), &unpack->start_ts));
         if (LF_ISSET(WT_CELL_TXN_START))
             WT_RET(__wt_vunpack_uint(&p, end == NULL ? 0 : WT_PTRDIFF(end, p), &unpack->start_txn));
+        if (LF_ISSET(WT_CELL_TS_DURABLE_START))
+            WT_RET(__wt_vunpack_uint(&p, end == NULL ? 0 : WT_PTRDIFF(end, p), &durable_ts_compat));
```

Comment:
This value is a difference, we need:
 `durable_ts_compat += unpack->start_ts;`

Reply:
We never actually use durable_ts_compat, that's why I didn't bother with the adjustment.